lib/ansible/plugins/lookup/password.py

```diff
@@ -149,13 +149,7 @@ def _random_password(length=DEFAULT_LENGTH, chars=C.DEFAULT_PASSWORD_CHARS):
     assert isinstance(chars, text_type), '%s (%s) is not a text_type' % (chars, type(chars))
 
     random_generator = random.SystemRandom()
-
-    password = []
-    while len(password) < length:
-        new_char = random_generator.choice(chars)
-        password.append(new_char)
-
-    return u''.join(password)
+    return u''.join(random_generator.choice(chars) for _ in range(length))
```

Comment:
Use ``dummy`` instead of ``_``  for dummy variable names (If we ever i18n the code, ``_`` will be used as an alias for gettext).